source/prerequisites/my-sql.txt

```diff
@@ -10,4 +10,92 @@ MySQL Migration Prerequisites
    :depth: 1
```

Comment:
[Info typing]

Sorry for not picking up on this before but could we make the [task page titles verb phrases](https://www.mongodb.com/docs/meta/style-guide/information-types/prototypes/task/)? Something like "Configure MySQL for Sync Jobs"?

Reply:
I think changing the title may be slightly misleading, there are not always configurations changes to be made. It is dependent on each cluster and deployment. So I will leave these as is for now, under that causality.